client/components/web-preview/index.jsx

```diff
@@ -1,3 +1,5 @@
+/** @ssr-ready **/
+
```

Comment:
> Kind of weird to mark this file as @ssr-ready since it contains references to window and document

Yeah, I can't say I'm totally happy with this situation. Seems like it could be fragile. Let's see how it goes...